src/controllers/index.controller.js

```diff
@@ -177,6 +178,10 @@ function start(req, res) {
   }
 
   const wallet = new HathorWallet(walletConfig);
+
+  // XXX: subscribe to wallet events with notificationBus
+  notificationBus.subscribeHathorWallet(req.body['wallet-id'], wallet);
```

Comment:
The `subscribeHathorWallet` method does not create listeners on the `HathorEvents` object, but on the `HathorWallet` and `Connection` event emitters.
So yes, destroying the event emitters with the listeners will be enough to remove all listeners.

This would change if we were to add listeners on the `HathorEvents` object but it's not on the scope of this project.